src/System.Private.CoreLib/shared/System/DateTimeOffset.cs

```diff
@@ -553,8 +553,8 @@ void IDeserializationCallback.OnDeserialization(object sender)
         {
             try
             {
-                _offsetMinutes = ValidateOffset(Offset);
-                _dateTime = ValidateDate(ClockDateTime, Offset);
+                ValidateOffset(Offset);
```

Comment:
Throws if 
```csharp
new TimeSpan(0, _offsetMinutes, 0).Ticks % TimeSpan.TicksPerMinute != 0
```
and returns
```csharp
new TimeSpan(0, _offsetMinutes, 0).Ticks / TimeSpan.TicksPerMinute
```
So if the `_offsetMinutes` is valid it doesn't change it, and if invalid it throws (are other checks); so the assignment is redundant (also prevents it being readonly)